components/confirm_modal.jsx

```diff
@@ -107,7 +109,12 @@ export default class ConfirmModal extends React.Component {
 
     handleKeypress = (e) => {
         if (e.key === 'Enter' && this.props.show) {
-            this.handleConfirm();
+            const cancelButton = document.getElementById('cancelModalButton');
+            if (cancelButton && cancelButton.className.indexOf(A11yClassNames.ACTIVE) > -1) {
```

Comment:
Just want to confirm that we are okay tying this logic to the accessibility controller (`a11yController.js`)? Though I don't see it going away, if it ever did or something changed in how it functions, this could stop working. Should we maybe use a comparison like this instead?

```suggestion
            if (cancelButton && cancelButton === document.activeElement) {
```

Alternatively, if we are okay with relying on the `a11yController.jf`, then it might be a bit cleaner to use this:

```suggestion
            if (cancelButton && cancelButton.classList.contains(A11yClassNames.ACTIVE)) {
```

Thoughts?

Reply:
@deanwhillier Good call. I was originally using `document.activeElement`, but an unrelated issue was occurring at the time of using it.